WooCommerce/Classes/ViewRelated/Orders/Order Create/OrderFormViewController.swift

```diff
@@ -0,0 +1,85 @@
+import UIKit
+
+final class OrderFormViewController: UIViewController {
+
+    /// Main TableView.
+    ///
+    private lazy var tableView = UITableView(frame: .zero, style: .grouped)
+    private let dataSource: OrderFormDataSource = .init()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        configureNavigation()
+        configureTableView()
+    }
+
+    @objc func presentMoreOptionsActionSheet(_ sender: UIBarButtonItem) {
+        // TODO: show option to save a draft
+    }
+}
+
+// MARK: - User Interface Initialization
+//
+private extension OrderFormViewController {
+
+    func configureNavigation() {
+        title = Localization.newOrderTitle
+
+        navigationItem.rightBarButtonItem = createMoreOptionsBarButtonItem()
+    }
+
+    func createMoreOptionsBarButtonItem() -> UIBarButtonItem {
+        let moreButton = UIBarButtonItem(image: .moreImage,
+                                         style: .plain,
+                                         target: self,
+                                         action: #selector(presentMoreOptionsActionSheet(_:)))
+        moreButton.accessibilityLabel = Localization.moreOptionsAccessibilityLabel
+        return moreButton
+    }
+
+    func configureTableView() {
+        dataSource.registerTableViewCells(tableView)
+
+        tableView.delegate = self
+        tableView.dataSource = dataSource
+
+        view.backgroundColor = .listBackground
+        tableView.backgroundColor = .listBackground
+        tableView.sectionHeaderHeight = UITableView.automaticDimension
+        tableView.sectionFooterHeight = .leastNonzeroMagnitude
```

Comment:
This seems redundant since you already declared the delegate method, or I'm wrong?

Reply:
Good catch for header - I've added this one before implementing delegate later.
For footer it's still needed, I don't have its delegate method handled.